api/v1beta1/v1beta2_condition_consts.go

```diff
@@ -0,0 +1,262 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+// Conditions types that are used across different objects.
+const (
+	// AvailableV1beta2Condition reports if an object is available.
+	// Note: This condition type is defined to ensure consistent naming of conditions across objects.
+	// Please use object specific variants of this condition which provides more details for each context where
+	// the same condition type exists.
+	AvailableV1beta2Condition = "Available"
```

Comment:
```suggestion
	AvailableV1Beta2Condition = "Available"
```
I think, please check for other occurences

Reply:
Fixed all around